doc/language-support.xml

```diff
@@ -196,12 +196,12 @@ you need it.</para>
 
 <para>
   Currently supported interpreters are <varname>python26</varname>, <varname>python27</varname>,
-  <varname>python32</varname>, <varname>python33</varname>, <varname>python34</varname>
+  <varname>python33</varname>, <varname>python34</varname>, <varname>python35</varname>
```

Comment:
We follow upstream which will stop receiving security updates in Feb 2016 https://www.python.org/dev/peps/pep-0392/#lifespan

You can still use 3.2 packages, we just don't officially maintain them and provide binaries for it. 

3.2 was a big pain to maintain so the reason people were still doing that was Ubuntu TLS 12.04 that got replaced with 14.04.